### PR TITLE
Add new project hook command

### PR DESCRIPTION
**Pull request by depwaswho94**

### CODE REVIEW

File: cmd/flags_definers.go [hunk 1]
```diff
@@ -196,6 +196,28 @@ func addNewProjectFlags(cmd *cobra.Command) {
...
 }
 
+// addNewProjectHookFlags add the required flags for creating a new project hook
+// Flag usage reference: https://docs.gitlab.com/ce/api/projects.html#add-project-hook
+func addNewProjectHookFlags(cmd *cobra.Command) {
+	addNewProjectHookEditProjectHookFlags(cmd)
+}
+
+func addNewProjectHookEditProjectHookFlags(cmd *cobra.Command) {
+	cmd.Flags().String("url", "", "The hook URL")
```
Comment (bzon) [inline cmd/flags_definers.go]: The url should be required right?

File: cmd/new_project_hook.go [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	}
+	hook, errr := newProjectHook(name, opts)
+	if errr != nil {
+		return errr
+	}
+	printProjectHooksOut(cmd, hook)
+	return nil
+}
+
+func newProjectHook(pid string, opts *gitlab.AddProjectHookOptions) (*gitlab.ProjectHook, error) {
```
Comment (bzon) [inline cmd/new_project_hook.go]: Rename `pid` to `project` instead. This is the pattern that we used in other command functions.

File: cmd/new_project_hook_test.go [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+				"merge-requests-events": 			"true",
+				"tag-push-events": 					"true",
+				"note-events": 						"false",
+				"job-events": 						"false",
+				"pipeline-events": 					"false",
+				"wiki-page-events": 				"false",
+				"enable-ssl-verification": 			"false",
+				"token": 							"LxryAgAdBoGXdOVPshHD",
+			},
+			args:   []string{"4"},
```
Comment (bzon) [inline cmd/new_project_hook_test.go]: Please add a test case using the project path as args.

File: cmd/flags_definers.go (hunk 1 above)
Comment (depwaswho94) [reply cmd/flags_definers.go]: Done

File: cmd/new_project_hook.go (hunk 2 above)
Comment (depwaswho94) [reply cmd/new_project_hook.go]: Done

File: cmd/new_project_hook_test.go (hunk 3 above)
Comment (depwaswho94) [reply cmd/new_project_hook_test.go]: Done

**Pull request merged**